### ISSUE TITLE
Force setting does not force?

### ISSUE THREAD

**Issue by Jos512** [bep/s3deploy#44]
Perhaps I misunderstand, but I read this in the README:

>   -force
    	upload even if the etags match

I run s3deploy 2.3.0 on CircleCI with this command:

~~~
s3deploy -bucket mysite -region eu-central-1 -source public -key $AWS_KEY -secret $AWS_SECRET -config .s3deploy.yml -public-access true -force
~~~

I added the `-force` command to the last of that command. This is however the output:

~~~
s3deploy 2.3.0, commit ed74ea6018a0859b89806a809368ec812e9cc9dd, built at 2019-01-01T14:01:40Z
404.html (ETag) ↑ about/index.html (size) ↑ contact/index.html (size) ↑ csharp/atom.xml (size) ↑ csharp/computer-drive/check-ready-drive/index.html (size) ↑ csharp/computer-drive/difference-total-free-space/index.html (size) ↑ csharp/computer-drive/drive-free-space/index.html (size) ↑ csharp/computer-drive/drive-root-directory/index.html (size) ↑ csharp/computer-drive/filter-computer-drives/index.html (size) ↑ csharp/computer-drive/get-drive-info/index.html (size) ↑ csharp/computer-drive/inaccessible-drives/index.html (size) ↑ csharp/computer-drive/index.html (size) 

[... removed the hundreds of HTML files for brevity ]

Total in 2.93 seconds
Deleted 0 of 0, uploaded 306, skipped 1006 (23% changed)
~~~

I don't get why `s3deploy` still skipped 1,006 files when I used `force`. Am I using the feature wrong?

**Comment by bep** [comment]
That is odd ... I just tested with one of my test sites and got the expected output:

```bash
s3deploy 2.3.0, commit ed74ea6018a0859b89806a809368ec812e9cc9dd, built at 2019-01-01T14:01:40Z
b1.js (force) ↑ b2.js (force) ↑ bbundle.js (force) ↑ categories/index.html (force) ↑ categories/index.xml (force) ↑ fonts/open-iconic.eot (force) ↑ fonts/open-iconic.otf (force) ↑ fonts/open-iconic.svg (force) ↑ fonts/open-iconic.ttf (force) ↑ fonts/open-iconic.woff (force) ↑ images/sunset_hu59e56ffff1bc1d8d122b1403d34e039f_90587_300x200_fill_q75_box_smart1.jpg (force) ↑ index.html (force) ↑ index.xml (force) ↑ post/hello-scss/index.html (force) ↑ post/index.html (force) ↑ post/index.xml (force) ↑ sitemap.xml (force) ↑ styles/bundle1.min.css (force) ↑ styles/main.min.ccc7b87f7432dde1870e402a534c9ea62c11dcf3595bc3c68b52e3d185e8d1b1.css (force) ↑ tags/index.html (force) ↑ tags/index.xml (force) ↑ Create CloudFront invalidation request for [/hugoscss/*]
Total in 2.85 seconds
Deleted 0 of 0, uploaded 21, skipped 0 (100% changed)
```

I don't see anything obviously wrong with your command, but I would suspect that the `force` flag for some reason isn't set.

**Comment by Jos512** [comment]
Thanks for the quick reply!

I see where I went wrong; those 1,006 files that weren't overwritten by `s3deploy` were uploaded by another AWS IAM user (other key and secret than I use with s3deploy). So it either couldn't read those from the bucket or overwrite them.

I've manually cleared my bucket by hand first, and then had s3deploy upload all files. Now `force` works as intended. :slightly_smiling_face: :
